reproject/utils.py

```diff
@@ -27,6 +27,11 @@ def _dask_to_numpy_memmap(dask_array):
     memmap array.
     """
 
+    # Sometimes compute() has to be called twice to return a Numpy array,
+    # so we need to check here if this is the case and call the first compute()
+    if isinstance(dask_array.ravel()[0].compute(), da.Array):
+        dask_array = dask_array.compute()
```

Comment:
This is to avoid issues seen with @Cadair's data - interestingly a similar issue was seen with a different dataset in glue (https://github.com/glue-viz/glue/pull/2399) so maybe this is a common issue, hence why it might be worth having the workaround here.